sys/include/crypto/ciphers.h

```diff
@@ -96,8 +107,10 @@ typedef struct cipher_interface_st {
 
 typedef const cipher_interface_t *cipher_id_t;
 
+/* @deprecated */
```

Comment:
See comment above. I don't think we need to duplicate the deprecation hint, but you have to check which file is better.  Probably the header file, but `CIPHER_AES` is allocated in aes.c ...